ctru-rs/examples/ir-user-circle-pad-pro.rs

```diff
@@ -0,0 +1,259 @@
+//! A demo of using the ir:USER service to connect to the Circle Pad Pro.
```

Comment:
The header comment should be changed to more closely resemble the other example headers (Name of the example followed by a simple description).